build-docker-images.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+print_something() {
+  something=$1
+
+  echo "##############################################################"
+  echo ""
+  echo $something
+  echo ""
+  echo "##############################################################"
+}
+
+docker_build() {
+  image=$1
+  tag=$2
+  build_arg=$3
+
+  print_something "Build image kuzzleio/$image:$tag from kuzzle-containers/$image/Dockerfile"
+
+  docker build -t kuzzleio/$image:$tag -f kuzzle-containers/$image/Dockerfile --build-arg $build_arg kuzzle-containers/$image
+}
+
+docker_tag() {
+  image=$1
+  tag=$2
+
+  print_something "Tag image kuzzleio/$image with $tag"
+
+  docker tag kuzzleio/$image kuzzleio/$image:$tag
+}
+
+docker_push() {
+  image=$1
+  tag=$2
+
+  print_something "Push image kuzzleio/$image:$tag to Dockerhub"
+
+  docker push kuzzleio/$image:$tag
+}
+
+git_clone() {
+  git clone --depth 1 https://github.com/kuzzleio/kuzzle-containers
+  # cp ../kuzzle-containers .
+  # use a test docker repository for now
+  mv kuzzle-containers/kuzzle kuzzle-containers/kuzzle-test
+}
+
+if [ -z "$TRAVIS_BRANCH" ]; then
+  echo "TRAVIS_BRANCH not found"
+  exit 1
+fi
+
+
+if [ "$TRAVIS_BRANCH" == "1.x" ]; then
```

Comment:
Does this mean we'll have only 1 `develop` tag? What happens when we start working on v2?

Reply:
When we start working on `2.x`, we will have to rethink our `develop` tag I think.  
Maybe we can simply use `kuzzleio/kuzzle:1.x` and `kuzzleio/kuzzle:2.x` tags.

Reply:
I think we need a workshop on this subject.  
Between Kuzzle, the SDK, the Documentation, we are going to handle a lot of different versions at the same time

Reply:
@ballinette > I don't see why we should have a generic tag name for development, for the same reason I'm resolutely against having one on github: because of version collisions, especially during transition phases.
Since we're going to maintain multiple major versions, I think it'd be best to tag our docker images accordingly, for instance `<major version number>-develop` (this comes off the top of my head, I didn't think that through yet). 
Moreover, these tags could be dynamically generated, and having explicit tag names would also lower the probability of human errors.

Reply:
Many projects use a `develop` name for current development branch and `x.y` for old LTS versions, and they don't seem to have issue with that.
Many other use only numeric version branches also.

I really don't care on which option we choose, we just need a quick workshop to make a decision and be consistent... but it's not an issue for this PR anyway.

Reply:
But I also like your proposal with `<major version number>-develop` ;)

Reply:
I think we need to formalize all that with the team before applying it in the naming of the tags of our images.  
Otherwise I also like @scottinet proposal `<major version number>-develop `